docs_2.0/manual_EN/4.ngql-guide/3.data-types.md/9.type-conversion.md

```diff
@@ -0,0 +1,23 @@
+# Type Conversion
```

Comment:
is implicit type conversion supported in 2.0??

Reply:
not for now